_rules/html-lang-valid-bf051a.md

```diff
@@ -19,11 +19,11 @@ authors:
 
 ## Applicability
 
-The root element of the page, if it is an `html` element with a `lang` and/or `xml:lang` attribute that is not empty ("").
+The root element of the page, if it is an `html` element with a `lang` and/or `xml:lang` attribute that is not [empty](#empty).
 
 ## Expectation
 
-The `lang` and `xml:lang` attributes have a [valid language subtag](#valid-language-subtag) if the attribute is [non-empty](#non-empty).
+The `lang` and `xml:lang` attributes have a [valid language subtag](#valid-language-subtag) if the attribute is not [empty](#empty).
```

Comment:
`empty` isn't a term in the glossary either. https://act-rules.github.io/glossary/

Reply:
I’ve added empty to the glossary inside this branch: 
https://github.com/act-rules/act-rules.github.io/pull/634/commits/d35e16878f7022dac6d2261d4de6227b3f9aa4e2

Reply:
@WilcoFiers Actually, there's something deeper with our use of `empty`. I see 69 uses of `empty` or variations (`non-empty`, `not empty`, …) in our rules and only the 2 usages in that rule tried to link to the glossary, the rest being either `empty ("")` or just `empty`… And such a discrepancy doesn’t make sense :-(

And there's also some other places that refers to `not only whitespace` which certainly means that I've got the wrong definition of `empty` over there :-((

Anyway, I feel we should unify the way we use and explain `empty` in the rules: either the inline definition `empty ("")` or linking to some glossary entry. We should not mix the two ways to do it. And we should probably also give the explanation every single time we use the word (or at least once per rule).

Reply:
@WilcoFiers I've removed the incorrect glossary entry that I added previously, and I've updated this rule to use `empty ("")` like we do all over the place.